### DISCUSSION TITLE
error finding lib file

### DISCUSSION THREAD

**Discussion by cfritze64** [johnlindquist/kit#435] [Errors] [answered]
beta 58 new error?
The offending call in my scripts is like this one:

let { getActiveScreen, setActiveAppBounds } = await kit("desktop")

desktop.js is in /Users/BikeFixe/.kit/lib as I think it was before. 

Error reported by Kit:

/Users/BikeFixe/.kit/scripts/desktop.js not found
Error: /Users/BikeFixe/.kit/scripts/desktop.js not found
    at resolveToScriptPath (file:///Users/BikeFixe/.kit/core/util.js:83:19)
    at global.run (file:///Users/BikeFixe/.kit/api/kit.js:181:26)
    at kit (file:///Users/BikeFixe/.kit/api/kit.js:320:19)
    at file:///Users/BikeFixe/.kenv/scripts/right-app.js?uuid=ff68881b-7861-4442-bbe3-6b654304237a:6:53
    at ModuleJob.run (node:internal/modules/esm/module_job:154:23)
    at async Loader.import (node:internal/modules/esm/loader:166:24)
    at async global.attemptImport (file:///Users/BikeFixe/.kit/api/kit.js:51:16)
    at async file:///Users/BikeFixe/.kit/cli/app-run.js?uuid=e1c46267-8240-46fc-a78b-8d8124977f07:77:9

**Comment by johnlindquist** [comment, marked as answer]
You can delete the line:

`let { getActiveScreen, setActiveAppBounds } = await kit("desktop")`

`getActiveScreen` and `setActiveAppBounds` are globals now. So that line is unnecessary. Apologies for not making that deprecating that more smoothly

**Reply by johnlindquist** [reply]
Sidenote: the `kit` function will now be used in webhooks, etc for running scripts located in a `scripts` dir.

That's why it's getting the "Can't find `.kit/scripts/desktop.js` because it's trying to run a script named "desktop" in the `.kit/scripts` dir.

**Reply by cfritze64** [reply]
Cool, and thanks!
